dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/RegexUtils.java

```diff
@@ -32,14 +32,15 @@
 
     private static final String LINUX_USERNAME_PATTERN = "[a-z_][a-z\\d_]{0,30}";
 
+    private static final String WINDOWS_USERNAME_PATTERN = "[a-z_][a-z\\d_]{0,30}";
```

Comment:
`WINDOWS_USERNAME_PATTERN` is just same with `LINUX_USERNAME_PATTERN`, and not for Windows